### DISCUSSION TITLE
Add a time series plot of 'Current Consumption' stat

### DISCUSSION THREAD

**Discussion by radh21301** [jheredianet/Teslamate-CustomGrafanaDashboards#105] [Ideas]
In the 'Current Drive View', there is a box showing the current consumption in wh/mi or wh/km. It is a single number. It would be awesome if we can see a plot as that value changes over time, just like the plot on the 'energy' app on Tesla.

<img width="223" alt="image" src="https://github.com/user-attachments/assets/eaf432f8-70c5-49fd-a652-c64a64dda021">

**Comment by swiffer** [comment]
isn't that what's covered by the power graph ? (while beeing a different unit, but we should be able to calculate that based on current power and speed, right?)

**Reply by swiffer** [reply]
```sql
	case
		when avg(speed) < 5 then 0
	  else coalesce(avg(power) * 1000 / nullif(avg(convert_km(speed::numeric, '$length_unit')), 0), 0)
	end as consumption_kwh_$length_unit,
```

guess adding that one would help, right?

![grafik](https://github.com/user-attachments/assets/a88ca6b7-d3df-4aa4-a25e-d264a2783c17)

**Comment by jheredianet** [comment]
@radh21301 Are you sure you are using the latest version of the dashboard?
Actually, it looks like this:

![image](https://github.com/user-attachments/assets/aacb5c31-cedd-4414-8ed8-c590e6cdbea3)

There you already have the plot you are asking for (In the main timeline) I think there is no need to put it in the gross consumption.
